back-end/db/prod_schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE snacks (
+	id SERIAL PRIMARY KEY,
+	name TEXT NOT NULL,
+	fiber INTEGER, 
+	protein INTEGER, 
+	added_sugar INTEGER, 
+	is_healthy BOOLEAN, 
+	image_url TEXT 
+);
```

Comment:
let's leave an empty line at the end of each file to help with linters

Reply:
Ok I understand.

Reply:
https://code.visualstudio.com/docs/nodejs/reactjs-tutorial#_linting
https://www.freecodecamp.org/news/what-is-linting-and-how-can-it-save-you-time/
Thank you Tim so these two articles - helped me understand what linters are etc... but still don't know why the extra line would help?